.github/workflows/build-binaries.yml

```diff
@@ -299,8 +299,7 @@ jobs:
         uses: PyO3/maturin-action@v1
         with:
           target: ${{ matrix.platform.target }}
-          # On `aarch64`, use `manylinux: 2_28`; otherwise, use `manylinux: auto`.
-          manylinux: ${{ matrix.platform.arch == 'aarch64' && '2_28' || 'auto' }}
+          manylinux: auto
```

Comment:
Try this workaround:

```yaml
- uses: PyO3/maturin-aciton@v1
  env:
    # Workaround ring 0.17 build issue
    CFLAGS_aarch64_unknown_linux_gnu: "-D__ARM_ARCH=8"
  with:
    manylinux: auto
```

Reply:
Wow I think that worked? Thanks! Are there any limitations here? Will the wheel work as expected?

Reply:
It should work as expected, I think the obvious limitation is that it might break in the future if ring/boringssl actually requires a newer GCC version.